### ISSUE TITLE
Issues with Training

### ISSUE THREAD

**Issue by arnavm30** [kaidic/GST#6]
Hi,

I'm trying to reproduce the results of the paper on the MalNetTiny dataset, but the model does not seem to be training properly, with the loss and accuracy not improving, even after modifying some hyperparameters. I think there could potentially be an issue in the training pipeline as I'm getting the following warning:

```torchmetrics/utilities/prints.py:36: UserWarning: No positive samples in targets, true positive value should be meaningless. Returning zero tensor in true positive score```

Could an author confirm that this public code should work?

Thanks!
